docs/usage.md

```diff
@@ -47,7 +47,7 @@ You can avoid running any startup scripts by adding `class="mejs__player"` to th
 ### Vanilla JavaScript
 ```html
 <script>
-	var player = new MediaElementPlayer('player', {
+	var player = new MediaElementPlayer(document.querySelector('#mediaplayer'), {
```

Comment:
The other option is also valid, since it works even on the main website, so I'll add a note about this other approach